src/tools/mat/qtxdg-mat.cpp

```diff
@@ -95,9 +95,27 @@ int main(int argc, char *argv[])
     // Find out the positional arguments.
     parser.parse(QCoreApplication::arguments());
     const QStringList args = parser.positionalArguments();
-    const QString command = args.isEmpty() ? QString() : args.first();
-    bool cmdFound = false;
+    parser.parse(QCoreApplication::arguments());
```

Comment:
Is there a reason to repeat it here? `parser.parse()` was called above, before a `const` method.

Reply:
No, just a leftover typo. Fixed.
Thanks.

Reply:
No, just a leftover typo. Fixed.
Thanks.